src/Native/jitinterface/jitwrapper.cpp

```diff
@@ -78,3 +89,12 @@ DLL_EXPORT int JitCompileMethod(
 
     return 1;
 }
+
+DLL_EXPORT unsigned GetMaxIntrinsicSIMDVectorLength(
+    Jit * pJit,
+    CORJIT_FLAGS flags)
+{
+    // BUGBUG: the signature on the Jit side is actually "pass by value", but for reasons that escape me,
```

Comment:
Again, couldn't figure out why the ABI doesn't match. The flags that come from the managed side look alright in the debugger, but become broken on the RyuJIT side. Of course it can't be checked in like this, but I already spent too much time on this and need a fresh pair of eyes to have a look.